app/src/main/java/com/siri/RearrangeScreens/MainActivity.kt

```diff
@@ -1,23 +1,26 @@
 package com.siri.RearrangeScreens
 
-import android.app.Activity
 import android.content.Intent
 import android.os.Bundle
+import android.util.Log
 import android.widget.Button
 import androidx.appcompat.app.AppCompatActivity
 import com.siri.RearrangeScreensSDK.ScreenUtils
 
 class MainActivity : AppCompatActivity() {
-
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        var screenNames: Array<String> = arrayOf("Screen1", "Screen2", "Screen3")
+        ScreenUtils.takeScreenNames(this@MainActivity, screenNames)
         setContentView(R.layout.activity_main)
-        ScreenUtils.takeScreenNames(this@MainActivity,"Screen1")
+        val str_Array: Unit = ScreenUtils.returnScreenOrder()
+        Log.d("received screen order ", "Recieved screen order is :" + (str_Array))
```

Comment:
I don't think `returnScreenOrder` helps you check `takeScreenNames` because your `returnScreenOrder` function returns hard-coded values instead of those that were passed into the `takeScreenNames` function:
```
    fun returnScreenOrder() {
         var screenOrderList: Array<String> = arrayOf("Screen3", "Screen1", "Screen2");
     }
```

So seems like either you meant to not hard-code the values in `returnScreenOrder` or you else why call `returnScreenOrder` here?